bundles/org.openhab.binding.worxlandroid/src/main/resources/landroid_error_de.map

```diff
@@ -0,0 +1,20 @@
+UNKNOWN=unknown
```

Comment:
file ends with `de` suggesting it is a german map (or am i mistaken?)
Better to change it to `en`

Same for the order map file.

Reply:
Well spotted. Corrected